examples/plotting/imageanimatorwcs.py

```diff
@@ -77,7 +77,12 @@
 # of ``wcs_anim`` allows us to set various properties.
 # We assign them to a easy to remember variable name.
 # Note the order of assignment out from ``wcs_anim.axes.coords``.
-time, solar_y, solar_x = wcs_anim.axes.coords
+
```

Comment:
We shouldn't be doing this in the example gallery, we should let astropydev fail.

Reply:
Fine.